contracts/ibc-reflect/src/contract.rs

```diff
@@ -355,9 +355,7 @@ mod tests {
     }
 
     fn fake_events(reflect_addr: &str) -> Vec<Event> {
-        let event = Event::new("message").add_attributes(vec![
-            attr("module", "wasm"),
-            attr("signer", MOCK_CONTRACT_ADDR),
```

Comment:
Were these unnecessary?

Reply:
The events changed. We no longer emit message events (or no longer expose those to the contracts), However, we have instantiate/execute/migrate events with a subset of the event types.

I wrote this to document the new behavior, which we need to model/mock: https://github.com/CosmWasm/wasmd/blob/master/EVENTS.md (and we are updating wasmd to it right now)